promptsource/templates/oskarvanderwal/crows_pairs_multilingual/english/templates.yaml

```diff
@@ -0,0 +1,110 @@
+dataset: oskarvanderwal/crows_pairs_multilingual
```

Comment:
I have never added a dataset to the main datasets before, so I am not sure whether I can fix that today. Another solution would be to add it to [the organization of the bias evaluation WG](https://huggingface.co/BigScienceBiasEval), since we also intend to submit another dataset to promptsource from there.

Reply:
I have moved the dataset to `BigScienceBiasEval/crows_pairs_multilingual` for now. Let me know if that's a good solution